src/main/features/claimant-response/helpers/settlementHelper.ts

```diff
@@ -44,18 +44,18 @@ export function prepareDefendantPartyStatement (claim: Claim, draft: DraftClaima
 export function prepareDefendantOffer (claim: Claim, draft: DraftClaimantResponse): Offer {
   const response: FullAdmissionResponse | PartialAdmissionResponse = claim.response as FullAdmissionResponse | PartialAdmissionResponse
 
+  const amount = response.responseType === ResponseType.PART_ADMISSION ? NumberFormatter.formatMoney(response.amount) : NumberFormatter.formatMoney(claim.totalAmountTillToday)
```

Comment:
i think you can use AmountHelper class here...

Reply:
Addressed this change.